SI/Sem.02/Pract.02/task_1_p02.cpp

```diff
@@ -0,0 +1,19 @@
+
+#include <iostream>
+
+int main()
+{
+    int year;
+    std::cout << "Enter a year: ";
+    std::cin >> year;
+    if ((0 == year % 4 )||( 0 != year % 100 && 0 == year % 400) )
```

Comment:
year % 4 == 0 is more readable way of writing code
Because you first write the expression and then you write what you expect to be equal to